src/main/java/net/openhft/chronicle/core/ChronicleInit.java

```diff
@@ -46,6 +52,20 @@ private ChronicleInit() {
                 ex.printStackTrace();
             }
         }
+
+        try {
+            ServiceLoader<ChronicleInitRunnable> runnableLoader = ServiceLoader.load(ChronicleInitRunnable.class);
+
+            for (Runnable runnable : runnableLoader) {
+                try {
+                    runnable.run();
+                } catch (Exception ex) {
+                    ex.printStackTrace();
```

Comment:
Sonar will complain about these

Reply:
What's the alternative here? No logging subsystem is expected to be up at this point.

Reply:
Fair point!